Makefile

```diff
@@ -231,8 +231,8 @@ benchmark:
 ###############################################################################
 
 lint:
-	golangci-lint run
-	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" | xargs gofmt -d -s
```

Comment:
Should we keep the find for auto `gofmt`'ing?

Reply:
Shouldn't "lint" be idempotent and "stateless"? Like shouldn't the `format` target do formatting?

Reply:
oh, I actually didn't realize `make format` existed. I think your totally right!